### PR TITLE
config: introduce memtx.sort_threads option

### PR DESCRIPTION
**Pull request by ImeevMA**
Part of #8861

### CODE REVIEW

File: changelogs/unreleased/gh-8861-memtx.sort_threads-option.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## feature/config
+
+* Option `memtx.sort_threads` is now supported (gh-8861).
```
Comment (xuniq) [inline changelogs/unreleased/gh-8861-memtx.sort_threads-option.md:3]:
```suggestion
* The `memtx.sort_threads` option is now supported (gh-8861).
```

**Review by xuniq — APPROVED** [review]

File: changelogs/unreleased/gh-8861-memtx.sort_threads-option.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## feature/config
```
Comment (Totktonada) [inline changelogs/unreleased/gh-8861-memtx.sort_threads-option.md:1]: Needs rebase after PR #8877 and removing of the option from the ignore list in the `test_box_cfg_coverage` test case.

File: src/box/lua/config/instance_config.lua [hunk 3]
```diff
@@ -715,6 +715,12 @@ return schema.new('instance_config', schema.record({
             box_cfg = 'memtx_max_tuple_size',
             default = 1024 * 1024,
         }),
+        sort_threads = schema.scalar({
+            type = 'number',
```
Comment (Totktonada) [inline src/box/lua/config/instance_config.lua:719]: I doubt that a fractional part has any meaning here. So, I would mark it as `'integer'`.

**Review by Totktonada — APPROVED** [review]

File: changelogs/unreleased/gh-8861-memtx.sort_threads-option.md (hunk 2 above)
Comment (ImeevMA) [reply changelogs/unreleased/gh-8861-memtx.sort_threads-option.md:1]: Done.

File: src/box/lua/config/instance_config.lua (hunk 3 above)
Comment (ImeevMA) [reply src/box/lua/config/instance_config.lua]: Done.

**Review by Totktonada — APPROVED** [review]

**Pull request merged**